### PR TITLE
First query and query test

### PR DESCRIPTION
**Pull request by iPhatty**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "express-handlebars": "^3.0.0",
     "pg": "^7.4.3",
     "pg-promise": "^8.4.3",
-    "serve-favicon": "^2.5.0"
+    "serve-favicon": "^2.5.0",
+    "url": "^0.11.0"
```
Comment (haydnba) [inline package.json:42]: get rid of this in the future!

File: src/model/database/db_build.js [hunk 2]
```diff
@@ -8,12 +8,6 @@ const sql = file => QueryFile(path.join(__dirname, file), { minify: true });
...
 
 // database building for test
-const runDbBuild = (cb) => {
-  db.query(build)
-    .then((res) => {
-      cb(null, res);
-    })
-    .catch(error => cb(error));
-};
+const runDbBuild = () => db.query(build);
```
Comment (haydnba) [inline src/model/database/db_build.js:11]: nice refactoring! 👮

File: src/test/test.js [hunk 3]
```diff
@@ -28,3 +33,22 @@ test('Test challenges route running', (t) => {
       t.end();
     });
 });
+
+// **********************************************
+// **************database tests******************
```
Comment (haydnba) [inline src/test/test.js:38]: good commenting Phat 🤓

File: src/test/test.js [hunk 4]
```diff
@@ -28,3 +33,22 @@ test('Test challenges route running', (t) => {
...
+// **********************************************
+
+test('Test for the first row of challenges query', (t) => {
+  const expected = 'Morning Hydration';
+  runDbBuild()
+    .then((res) => {
+      t.ok(res);
+      getAllChallenges()
```
Comment (i2xzy) [inline src/test/test.js]: try return getAllChallenges() and put the then outside the brackets

**Review by haydnba — APPROVED** [review]
Nice Work 🐩

**Pull request merged**